dot/config.go

```diff
@@ -24,21 +24,22 @@ import (
 	"reflect"
 	"unicode"
 
+	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/node/gssmr"
 	"github.com/ChainSafe/gossamer/node/ksmcc"
-
 	log "github.com/ChainSafe/log15"
 	"github.com/naoina/toml"
 )
 
 // Config is a collection of configurations throughout the system
 type Config struct {
-	Global  GlobalConfig  `toml:"global"`
-	Init    InitConfig    `toml:"init"`
-	Account AccountConfig `toml:"account"`
-	Core    CoreConfig    `toml:"core"`
-	Network NetworkConfig `toml:"network"`
-	RPC     RPCConfig     `toml:"rpc"`
+	Global  GlobalConfig     `toml:"global"`
+	Init    InitConfig       `toml:"init"`
+	Account AccountConfig    `toml:"account"`
+	Core    CoreConfig       `toml:"core"`
+	Network NetworkConfig    `toml:"network"`
+	RPC     RPCConfig        `toml:"rpc"`
+	System  types.SystemInfo `toml:"-"`
```

Comment:
does this exclude it from the config file?

Reply:
Yes.